be/src/column/chunk.h

```diff
@@ -245,6 +246,12 @@ class Chunk {
 
     bool has_const_column() const;
 
+    void materialize() {
```

Comment:
Maybe this `materialize` is too general, cause there're so many kinds of `materialize`.

How about make this name more specific to reflect its purpose ? Such as, `materialized_nullable` ?